engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbLanguage.java

```diff
@@ -29,15 +30,14 @@
  * provide context-switching facilities.
  */
 @TruffleLanguage.Registration(
-    id = EpbLanguage.ID,
+    id = ForeignLanguage.ID,
     name = "Enso Polyglot Bridge",
     characterMimeTypes = {EpbLanguage.MIME},
     internal = true,
     defaultMimeType = EpbLanguage.MIME,
     contextPolicy = TruffleLanguage.ContextPolicy.SHARED,
     services = Consumer.class)
 public class EpbLanguage extends TruffleLanguage<EpbContext> {
```

Comment:
Shouldn't this be renamed to `ForeignLanguage`  as well then? For now stuff is sometimes called epb and sometimes foreign language

Reply:
I am never sure `Epb` stands for, but we don't want to have two classes named `ForeignLanguage`

Reply:
ah right, the enum